cirq-web/cirq_ts/src/bloch_sphere/components/axes_test.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 The Cirq Developers
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {expect} from 'chai';
+import {Axes} from './axes';
+import {Line, LineDashedMaterial, Color} from 'three';
+
+describe('Axes', () => {
+  describe('defaults', () => {
+    const axes = new Axes(5);
+    const children = axes.children as Line[];
+
+    it('returns 3 Line objects', () => {
+      expect(children.length).to.equal(3);
+    });
+
+    it('returns the correct default colors for each line', () => {
+      const defaultColors = ['#1f51ff', '#ff3131', '#39ff14'];
+
+      children.forEach((el, index) => {
+        const material = el.material as LineDashedMaterial;
+        expect(material.color).to.eql(new Color(defaultColors[index]));
+      });
+    });
+
+    it('returns all lines with a constant linewidth (1.5)', () => {
+      children.forEach(el => {
+        const material = el.material as LineDashedMaterial;
+        expect(material.linewidth).to.equal(1.5);
+      });
+    });
+  });
+
+  describe('configurables', () => {
+    it('has configurable axis colors', () => {
+      const axes = new Axes(5, '#fff', '#fff', '#fff');
+      const children = axes.children as Line[];
+      for (const child of children) {
+        const material = child.material as LineDashedMaterial;
+        //.eql is deep equal, which can be used to compare objects
+        expect(material.color).to.eql(new Color(1, 1, 1));
```

Comment:
it's not obvious to me why `#fff` equals 1...Why is that?

Reply:
I'll add a comment for it, it's just saying that the color is white. RGB for white is (1, 1, 1)

Reply:
Is it though? I thought that would be (255, 255, 255)

Reply:
Here's how it works in three.js (https://threejs.org/docs/#api/en/math/Color.r). If we passed it in as a string `rgb(1, 1, 1)`, then I'd think that would be true, but in the `Color` constructor the parameters are red, green, and blue values in between 0 and 1.

Reply:
Ahhhhhhhh, got it. My bad.

Reply:
Also, github renders the color using `rgb(233,210,50)`? that's cool.